components/01-atoms/buttons/buttons.stories.js

```diff
@@ -29,9 +30,12 @@ storiesOf('Atoms/Buttons', module)
   .add('React button', () => (
     <Button>React Button</Button>
   ))
-  .add('Twig button', () =>
-    <div dangerouslySetInnerHTML={{__html: buttonTwig}}></div>
-  )
+  .add('Twig button', () => {
+    useEffect(() => {
+      require('./twig/button.js');
```

Comment:
Just a question: Why are we using the source file instead of the minified one?

Reply:
In Storybook, I thought it would be cool to use the unminified one for dev purposes.